sdk/subscribe_block.go

```diff
@@ -114,8 +113,9 @@ func (c *SubscribeService) getClient(add string) *ClientWebsocket {
 		return c.client
 	} else {
 		client, err := NewConnectWs(c.client.config.BaseURL.String(), *c.client.duration)
+
 		if err != nil {
-			fmt.Println(err)
+			panic(err)
```

Comment:
don't use panic - use error return

Reply:
fixed in https://github.com/proximax-storage/nem2-sdk-go/commit/e7f3dc739c27ae59cedd2846cc881ea4e1167569